src/Widgets/ThumbnailGalleryWidget/ThumbnailGalleryWidget.scss

```diff
@@ -99,7 +99,7 @@
   }
 }
 
-@media (max-width: $screen-sm-min) {
+@media (max-width: $screen-sm-max) {
```

Comment:
This changes the responsive behaviour.
If this was an off-by-one, it should now be `screen-xs-max`.

* [ ] FYI It looks good to me, but please make sure that the change is intended.

Reply:
This is actually an intended change by me. The design looks better to me, if we use all of `$screen-sm` and not just the lower bound.